### PR TITLE
Final cosmos pylint

### PR DESCRIPTION
**Pull request by bryevdv**
Here is the final set of changes to make Cosmos Pylint clean. I will submit a separate issue that describes the things that were ignored for now. 

cc @srinathnarayanan @annatisch

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py [hunk 1]
```diff
@@ -40,7 +40,9 @@ class CosmosClient:
     Use this client to configure and execute requests to the Azure Cosmos DB service.
     """
 
-    def __init__(self, url, auth, consistency_level="Session", connection_policy=None):
+    def __init__(
+        self, url, auth, consistency_level="Session", connection_policy=None
+    ):  # pylint: disable=missing-client-constructor-parameter-credentials,missing-client-constructor-parameter-kwargs,line-too-long
```
Comment (kristapratico) [inline sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py]: Just want to catch this so it's not a problem later - should be: disable=missing-client-constructor-parameter-**credential**. We had a discussion today in the office and decided that the credential param should be singular, not plural.

Comment (bryevdv) [reply sdk/cosmos/azure-cosmos/azure/cosmos/cosmos_client.py]: @kristapratico thank you I will fix this before merging

**Comment by bryevdv** [comment]
merging per OK from @srinathnarayanan

**Pull request merged**